quota/common/lib/quota-connect.js

```diff
@@ -79,12 +79,25 @@ function calcOptions(req, opts) {
   return options;
 }
 
-function applyQuota(self, options, resp, next) {
+function applyQuota(self, options, req, resp, next) {
```

Comment:
To avoid side effects and maintain backwards compatibility, make the new parameter (req) optional

Reply:
Done